### DISCUSSION TITLE
Does Autolink Reference support wildcard or regex?

### DISCUSSION THREAD

**Discussion by CWJie91** [community/community#145753] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

![image](https://github.com/user-attachments/assets/16afe6b8-7177-41a0-9b89-3dd518ae9f69)

Hi,

Good day to you.

My commit message is as this format add(123): add new feature and I want using that number to link to my jira issue
eg. https://abc-jira.youtube.com/browse/ABC-123.

Can I achieve using autolink reference that maybe support wildcard or regex?
If not , is there any other alternative?

**Comment by ujamX** [comment]
Hi,

Have a nice day too!

Currently, I understand that GitHub's automatic link referencing feature does not support wildcards or regular expressions for dynamic matching of commit messages.

However, I suspect that consistent use of a fixed prefix for issue numbers in commit messages, such as ABC-, will get you what you want.

**Reply by CWJie91** [reply]
Hi @ujamX,

Thanks for the replies.

My sample commit message will be as below:
1. `add(123): add new feature`
2. `change(456): change to bla bla`

Prefix can be `add(` or `change(`  and I tried and seem not working.

Is there any alternative? 

tq

**Reply by ujamX** [reply]
Hi @CWJie91 

There is no other alternative as we have not experienced the same situation.
It is expected that you will proceed sequentially after reviewing the official documents.
Best of luck

[docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/configuring-autolinks-to-reference-external-resources)
